pacman/operations/partition_algorithms/partition_and_place_partitioner.py

```diff
@@ -128,7 +129,8 @@ def _partition_vertex(
                 PartitionerMaximumSizeConstraint)
             for constraint in max_atom_constraints:
                 possible_max_atoms.append(constraint.size)
-        max_atoms_per_core = min(possible_max_atoms)
+
+        max_atoms_per_core = int(min(possible_max_atoms))
 
```

Comment:
this was because pycharm bitched wherever thats used, saying it was a iterable instead of a int. its just to shut it up.